import-export-cli/integration/envSpecific_test.go

```diff
@@ -29,7 +29,7 @@ import (
 	"github.com/wso2/product-apim-tooling/import-export-cli/integration/testutils"
 )
 
-func TestEnvironmentSpecificParamsEndpoint(t *testing.T) {
+func TestXyz(t *testing.T) {
```

Comment:
Why is this called TestXyz?

Reply:
This is already fixed via https://github.com/wso2/product-apim-tooling/pull/639/commits/1f2616e92cf314fb83d78c87ab8df456ed066a28